tempesta_fw/t/functional/long_body/test_long_response.py

```diff
@@ -0,0 +1,65 @@
+""" Testing for long body in response """
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2017-2018 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+import os
+import body_generator
+
+from testers import stress
+from helpers import tf_cfg, control, tempesta, remote
+
+class ResponseTestBase(stress.StressTest):
+    """ Test long response """
+    config = "cache 0;\n"
+    filename = "long.bin"
+    uri = "/" + filename
+    fullname = ""
+
+    def create_content(self, length):
+        """ Create content file """
+        content = body_generator.generate_body(length)
```

Comment:
It's good to test a lot of different kind of response. I think, we even need to go further: we need to prepare a back end configuration that will be capable to respond to POST, to serve a different type of resources, small and big ones. That will be a great enhancement for tests, isn't it?
Wee need to create a separate issue for that for one of the next milestones. What do you think?

Reply:
I think it's a good idea to make backend more configurable